lib/srv/db/common/errors.go

```diff
@@ -147,6 +149,32 @@ func ConvertConnectError(err error, sessionCtx *Session) error {
 	return trace.Wrap(err)
 }
 
+// createElastiCacheRedisAccessDeniedError creates an error with help message
+// to setup IAM auth for ElastiCache Redis.
+func createElastiCacheRedisAccessDeniedError(err error, sessionCtx *Session) error {
+	policy, getPolicyErr := dbiam.GetReadableAWSPolicyDocument(sessionCtx.Database)
```

Comment:
this requires #26648 ?

Reply:
It doesn't really require it since you can setup the permissions manually and without https://github.com/gravitational/teleport/pull/26648 the error message will just lack a policy document example, but I will be backporting https://github.com/gravitational/teleport/pull/26648 as well when it lands